functions/merge_sorted_lists.py

```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
```

Comment:
Is this the correct way to join two lists?

Reply:
I believe that is concatenation